_html/templates/modules/sponsors-categories.php

```diff
@@ -0,0 +1,114 @@
+                            <span class="name">Gold</span>
+                            <span class="quantity">4 sponsors</span>
+                        </h2>
+                        <div class="sponsors-categories--category--price">
+                            <span>Sponsorship value</span>
+                            <span>1,000 &euro;</span>
+                        </div>
+                    </div>
+                    <ul class="sponsors-categories--category--benefits">
+                        <li>Free tickets included: <span>6</span><br>(additional tickets can be bought online)</li>
+                        <li>Presentation stand at the event: <span>Gold Size</span></li>
+                        <li>Company presentation in the camp opening session: <span>3 min</span></li>
+                        <li>Logo added to the event's banners,printed on the event's material and featured on our website and in newsletters: <span>Gold Size</span></li>
+                        <li>Merchandising in the welcome pack</li>
+                    </ul>
+                    <div class="sponsors-categories--category--extras">
+                        <span>Coffee (1 sponsor)</span>
+                        <span>&nbsp;|&nbsp;</span>
+                        <span>Cookies (1 sponsor)</span>
+                        <span>&nbsp;|&nbsp;</span>
+                        <span>Snacks (1 sponsor)</span>
+                        <span>&nbsp;|&nbsp;</span>
+                        <span>Beer (1 sponsor)</span>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="l-row">
+            <div class="l-col l-col-2">
+                <div class="sponsors-categories--category silver">
+                    <div class="sponsors-categories--category--header">
+                        <h2 class="sponsors-categories--category--title">
+                            <span class="name">Silver</span>
+                            <span class="quantity">8 sponsors</span>
+                        </h2>
+                        <div class="sponsors-categories--category--price">
+                            <span>Sponsorship value</span>
+                            <span>500 &euro;</span>
+                        </div>
+                    </div>
+                    <ul class="sponsors-categories--category--benefits">
+                        <li>Free tickets included: <span>3</span><br>(additional tickets can be bought online)</li>
+                        <li>Presentation stand at the event: <span>Only Rollup</span></li>
+                        <li>Logo added to the event's banners,printed on the event's material and featured on our website and in newsletters: <span>Silver Size</span></li>
+                        <li>Merchandising in the welcome pack</li>
+                    </ul>
+                    <ul class="sponsors-categories--category--extras">
+                        <li></li>
```

Comment:
Do you need this empty li or the ones below?

Reply:
The block was not needed. I removed it.